### ISSUE TITLE
A Spirit Ally? Quest is Bugged

### ISSUE THREAD

**Issue by WindWarrior75** [dalaranwow/dalaran-wow#5696]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

Current behaviour: The Feralfen Totem has no area in which it can be placed.

Expected behaviour: The Feralfen Totem should be able to be placed at the bottom of the stairs.

Steps to reproduce the problem:

1. Accept "A Spirit Ally?" quest and acquire the Feralfen Totem
2. Go to the bottom of the stairs as instructed.
3. Right click on the Feralfen Totem in your inventory

Include proofs for this behaviour
![wowscrnshot_030519_162209](https://user-images.githubusercontent.com/47932086/53838361-de463a00-3f62-11e9-9f38-b0ee229e6bab.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills

The quest is named "A Spirit Ally?"
The item is named Feralfen Totem
You need to arrive at the Boha'mu Ruins Stairs.
You need to right click on the Feralfen Totem while near the bottom of the stairs (which doesn't seem to allow you to spawn the totem)

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
#2538
